bootstrap/kubeadm/api/v1alpha3/kubeadmbootstrapconfig_types.go

```diff
@@ -47,7 +47,7 @@ type KubeadmConfigSpec struct {
 
 	// Files specifies extra files to be passed to user_data upon creation.
 	// +optional
-	Files []File `json:"files,omitempty"`
+	Files []FileSource `json:"files,omitempty"`
```

Comment:
If we are going for this approach this way this is a breaking change, it should way for v1alpha4...

Reply:
fixed for backcompat